db/schema.rb

```diff
@@ -78,7 +78,7 @@
     t.integer "stream_id"
     t.integer "milliseconds", default: 0
     t.float "measured_value"
-    t.geometry "location", limit: {:srid=>4326, :type=>"geometry"}, null: false
+    t.geometry "location", limit: {:srid=>0, :type=>"geometry"}, null: false
```

Comment:
Hm, I'm wondering if this shouldn't be here, I'd say it shouldn't :D cc. @BartkiewiczM